lib/std/math/big/int_test.zig

```diff
@@ -2655,11 +2681,10 @@ test "big int popcount" {
     try popCountTest(&a, limb_size * 2 - 1, limb_size);
     try popCountTest(&a, limb_size * 2, limb_size + 1);
     try popCountTest(&a, limb_size * 2 + 1, limb_size + 2);
-    // TODO: These produce incorrect pop count for Mutable
-    // https://github.com/ziglang/zig/issues/13571
-    // try popCountTest(&a, limb_size * 2 + 2, limb_size + 3);
-    // try popCountTest(&a, limb_size * 2 + 3, limb_size + 4);
-    // try popCountTest(&a, limb_size * 2 + 4, limb_size + 5);
```

Comment:
I guess skipping for target wasm + adding `// TODO` with the link to the issue should be fine.

Reply:
Re-enabling these test cases shouldn't have triggered the wasm failure though, right? `popCountTest` doesn't create any u129+ values, it's purely working with big.Int and usize values.

Otherwise I must be misreading the code somewhere or misunderstanding the problem.